services/QuillLMS/config/application.rb

```diff
@@ -33,18 +33,6 @@ class Application < Rails::Application
       fog_directory: ENV.fetch('FOG_DIRECTORY', 'empirical-dev')
     }
 
-    # Custom directories with classes and modules you want to be autoloadable.
-    config.autoload_paths += %W(
-      #{config.root}/app/controllers/concerns
-      #{config.root}/app/models/validators
-      #{config.root}/app/queries/scorebook
-      #{config.root}/app/services
-      #{config.root}/app/services/analytics
-      #{config.root}/app/services/vitally_integration
-      #{config.root}/app/uploaders
-      #{config.root}/lib
-    )
```

Comment:
So excited that this is gone.

Reply:
Me too.  I think it was causing some hard to diagnose issues.